### PR TITLE
JENKINS-64357 Fix plugin on Jenkins >2.263

### PR DESCRIPTION
**Pull request by timja**
See https://issues.jenkins.io/browse/JENKINS-64357

and https://www.jenkins.io/doc/developer/views/table-to-div-migration/

Cleanup needed for me to compile this plugin (running java 11) is in https://github.com/jenkinsci/vsphere-cloud-plugin/pull/125

@pjdarton

### CODE REVIEW

File: src/main/resources/lib/vsphere/blockWrapper.jelly [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+                <d:invokeBody/>
+            </div>
+        </j:when>
+        <j:otherwise>
+            <table style="width:100%">
+                <d:invokeBody/>
+            </table>
+        </j:otherwise>
+    </j:choose>
+</j:jelly>
```
Comment (pjdarton) [inline src/main/resources/lib/vsphere/blockWrapper.jelly:16]: I presume that this is the core of the fix?
i.e. pre 2.264 it uses `table` and post it uses `div`?

...and does it do this automatically?  Is that what `test="${divBasedFormLayout}"` achieves?

**Review by pjdarton — COMMENTED** [review]
I think we need to resolve (PR 125)[https://github.com/jenkinsci/vsphere-cloud-plugin/pull/125#discussion_r546949281] before this one.
...but the only contentious stuff here (aside from #125) is down to my lack of understanding and request for explanations...

File: src/main/resources/lib/vsphere/blockWrapper.jelly (hunk 1 above)
Comment (timja) [reply src/main/resources/lib/vsphere/blockWrapper.jelly:16]: Yes it’s a property on set on >=2.264

**Comment by timja** [comment]
@pjdarton could this be released please?

**Comment by pjdarton** [comment]
Done.

**Pull request merged**